C/days of month.C

```diff
@@ -2,8 +2,8 @@
 #include<conio.h>
 void main()
```

Comment:
- You can improve this file more by assigning proper names to variables instead of a,b,i,j.
- add comments to explain the logic of the code
- rename the file to a better name

Reply:
Updated

Reply:
![image](https://user-images.githubusercontent.com/43691873/94777385-0ae16280-03e1-11eb-8aa3-2070fdde1281.png)

I can't see your changes!

Reply:
Updated...pls check it